spec/CloudCode.spec.js

```diff
@@ -1076,6 +1076,29 @@ describe('Cloud Code', () => {
       });
   });
 
+  /**
+   * Verifies that an afterSave hook throwing an exception
+   * will not prevent a successful save response from being returned
+   */
+  it('should succeed on afterSave exception', (done) => {
+    Parse.Cloud.afterSave("AfterSaveTestClass", function () {
+      throw "Exception";
+    });
+    const AfterSaveTestClass = Parse.Object.extend('AfterSaveTestClass');
+    const obj = new AfterSaveTestClass();
+    obj.save()
+      .then(function() {
+        // expected success
+        done();
+      })
+      .catch(function() {
+        // failed
+        fail("afterSave throwing an exception indicated that the save failed");
+        done();
+      })
+
```

Comment:
remove blank line.

also *could* write it:

```
obj.save().then(done, done.fail);
```

Reply:
Can do